### ISSUE TITLE
Reproduction of the results of the paper. 

### ISSUE THREAD

**Issue by XiudingCai** [TFboys-lzz/MPSCL#6]
I found that using the provided yaml file directly only reproduces half of the results in the paper. How can I reproduce the results reported in the paper? For example, the number of optimization iterations, batch size, etc.

**Comment by TFboys-lzz** [comment]
Hi Xiuding:
    Have you loaded the pre-trained model yet?

**Comment by XiudingCai** [comment]
您好，您的邮件我已收到。我会尽快处理，谢谢！Hola! Your e-mail has been received. I'll deal with it prompt. Thank you!

**Comment by XiudingCai** [comment]
Yes, I followed the readme file you provided strictly and I ran it 50k times as per the YAML file provided. But the reproduced experimental results are very low (about 24% for the dice metric) for both MPSCL and ADVENT, and I don't know what went wrong.

**Comment by TFboys-lzz** [comment]
Hi Xiuding:
    You can check if the images and labels are corresponding, and also check the performance of the model on the source domain

**Comment by XiudingCai** [comment]
Thanks for the quick reply, I printed the paths of the images and their corresponding labels and found that they were inconsistent, now I fixed the problem and started training again. Thanks again for your reply, it was very helpful!
